docs/apache-airflow/tutorial_taskflow_api.rst

```diff
@@ -181,6 +181,59 @@ and even a different python version to run your function.
 This option should allow for far greater flexibility for users who wish to keep their workflows more simple
 and pythonic.
 
+Creating Custom TaskFlow Decorators
+-----------------------------------
+
+As of Airflow 2.3, users can now integrate custom decorators into their provider packages and have those decorators
```

Comment:
```suggestion
As of Airflow 2.2, users can now integrate custom decorators into their provider packages and have those decorators
```

This will be in 2.2, no?